cchain/processors/es.py

```diff
@@ -74,7 +74,7 @@ def process_changes(self, changes_buffer):
 
         return processed_changes, last_seq
 
-    def persist_changes(self, processed_changes):
+    def persist_changes(self, processed_changes, **kwargs):
```

Comment:
I'd prefer to have it explicit to be honest, and describe in the docstring below